dist/messaging.js

```diff
@@ -286,6 +286,7 @@ const sendSMS = async function sendSMS(accessToken, userUuid, msg, fromPhoneNumb
  * @param {string} [userUUID="null userUUID"] - user uuid
  * @param {number} [offset=0] - pagination offest
  * @param {number} [limit=10] - pagination limit
+ * @param {boolean} [snooze=false] - snooze:true OR snooze:false to query conversations (default: false)
```

Comment:
This filter should have been isSnoozed or similar. We can keep it consistent with the API, but please update the comment to explain what it does.